docs/bucket/replication/README.md

```diff
@@ -18,8 +18,26 @@ mc admin bucket remote add myminio/srcbucket https://accessKey:secretKey@replica
 Role ARN = 'arn:minio:replication:us-east-1:c5be6b16-769d-432a-9ef1-4567081f3566:destbucket'
 ```
 
-Note that the admin needs *s3:GetReplicationConfigurationAction* permission on source cluster. The credential used at the destination requires *s3:ReplicateObject* permission. Once successfully created and authorized this generates a replication target ARN.  The command below lists all the currently authorized replication targets:
+Note that the admin user running the above command needs *s3:GetReplicationConfiguration* and *s3:GetBucketVersioning* permission on source cluster if admin user is not the default minio admin. The access credentials used at the destination requires *s3:ReplicateObject* permission.
 
+Minimal set of permissions needed on source bucket are
+```
+    "s3:GetReplicationConfiguration",
+    "s3:ListBucket",
+    "s3:GetBucketLocation",
+    "s3:GetBucketVersioning"
+```
+The access key configured to perform replication on the target cluster should have following permissions on destination bucket.Note that the "s3:GetBucketVersioning" permission is specific to the destbucket resource, and remaining permissions pertain to resource prefix.
+```
+    "s3:ReplicateTags",
+    "s3:GetObject",
+    "s3:GetObjectVersion",
+    "s3:GetObjectVersionTagging",
+    "s3:PutObject",
+    "s3:ReplicateObject"
+    "s3:GetBucketVersioning"
+```
+Once successfully created and authorized the `mc admin bucket remote add` command generates a replication target ARN.  The command below lists all the currently authorized replication targets:
```

Comment:
It would be worth to provide an entire JSON document with full examples @poornas